extension/fluentbitextension/README.md

```diff
@@ -0,0 +1,56 @@
+# FluentBit Extension
```

Comment:
Can we have a notice that this is an experimental extension and can be removed any time?

Reply:
Done.